.nuget/NuGet.targets

```diff
@@ -129,7 +129,7 @@
 

```

Comment:
I generally remove these old Nuget.targets.  This is just handled now in visual studio

Reply:
Maybe clean this up first as it's own PR

Reply:
Yeah, I'm going to decline this and create a new PR with the minimal changes since I've got a handle on what needs to be touched now.